.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
```

Comment:
looks like you've copied the file from a node project instead of a java one, "npm" is for Nodejs

Reply:
Yep. Looks like this value should be "maven".

See https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem and https://github.com/companieshouse/overseas-entities-api/blob/main/.github/dependabot.yml for an example.

Reply:
package-ecosystem changed to maven for java dependencies